src/System.Buffers/ref/Configurations.props

```diff
@@ -4,10 +4,11 @@
     <PackageConfigurations>
       netstandard;
       netstandard1.1;
-      uap10.0.16299;
+	    net45;
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations);
+      uap10.0.16299;
```

Comment:
Also, if you are flagging `uap10.0.16299` as inbox then you won't be needing this configuration any more.